lib/dstu2_resources.rb

```diff
@@ -305,6 +305,204 @@ module Resources
       ]
     }
 
+    DSTU2_CONDITION_BUNDLE ||= {
```

Comment:
Can you tweak your ids in here as well? As a rule it's probably not best to put actual ids in the samples especially if those id's belong to other people's data. It'll be more of an issue once we start having more writes, but even still the underlying data may change and no longer match what the samples show